packages/web-client/app/models/workflow/workflow-card.ts

```diff
@@ -25,8 +26,10 @@ export class WorkflowCard extends WorkflowPostable {
 
   @action onComplete() {
     this.isComplete = true;
+    this.workflow?.animatedWrapper?.startTestWaiter();
```

Comment:
This class should not know about animation or test waiters. This should be completely encapsulated in the AnimatedWorkflow. So this line should be something like `this.workflow?.onPostableComplete(this)` and then that should in turn do what needs to be done.